.github/workflows/deploy-lambdas.yml

```diff
@@ -0,0 +1,99 @@
+name: Deploy lambdas to S3 and ECR
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/deploy-lambdas.yml'
+      - 'lambdas/**'
+
+jobs:
+  deploy-lambda-s3:
+    strategy:
+      matrix:
+        path:
+          - access_counts
+          - es/indexer
+          - pkgevents
+          - pkgpush
+          - pkgselect
+          - preview
+          - s3hash
+          - s3select
+          - status_reports
+          - tabular_preview
+          - transcode
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build zip
+        run: |
+          BUILDER_IMAGE=quiltdata/lambda:build-3.8
+
+          docker pull "$BUILDER_IMAGE"
```

Comment:
well, you don't have to call pull here

Reply:
Ok. Does it mean I should also remove `--pull` from the `docker buildx build` command?